bin/nib-ls

```diff
@@ -22,6 +22,7 @@ import sys
 from math import ceil
 from optparse import OptionParser, Option
 from io import StringIO
+from collections import Counter
```

Comment:
Below you have:

```Python
counter = Counter()
counter.update(d.flatten())
# go through each entry and report only non-0 ones
row += ["@l" + " ".join("%g:%d" % (i, counter[i]) for i in sorted(counter))]
```

You could do something like:

```Python
row += ["@l" + " ".join("%g:%d" % (i, np.sum(d == i)) for i in np.unique(d))]
```

I suspect this would be slower than Counter, so take it for whatever it's worth.